vite.config.ts

```diff
@@ -18,6 +18,7 @@ export default defineConfig(({ mode }) => {
 
   return {
     build: {
+      chunkSizeWarningLimit: 900,
```

Comment:
* [ ] FMI We're just muting the warning without any comment why. The warning is there for a reason and it even gives instructions on how to handle it. Do we have a backlog item to fix the underlying problems?